node/src/Pos/Lrc/Worker.hs

```diff
@@ -197,7 +197,7 @@ lrcDo epoch consumers tip = tip <$ do
     whileAfterCrucial b = getEpochOrSlot b > crucial
     crucial = EpochOrSlot $ Right $ crucialSlot epoch
     withBlocksRolledBack blunds =
-        bracket_ (rollbackBlocksUnsafe blunds)
+        bracket_ (rollbackBlocksUnsafe (BypassSecurityCheck True) blunds)
```

Comment:
@pva701 you are overcomplicating. LRC almost always rolls back more than `k` blocks. The only exception is when chain quality is exactly 0.5 (in this case it rolls back exactly `k` blocks).